app/assets/src/components/common/DetailsSidebar/SampleDetailsMode/constants.js

```diff
@@ -51,3 +51,42 @@ export const PIPELINE_INFO_FIELDS = [
     key: "lastProcessedAt",
   },
 ];
+
+export const PIPELINE_RUN_STEPS = [
```

Comment:
Is there a way to avoid hardcoding these? If we change the steps in the pipeline in the future, they could go out of sync with this constant

Reply:
@tfrcarvalho How did Emily handle this for pipeline viz again? Did she just parse the underscores and re-capitalize to turn "validate_input_out" into "Validate Input"? That would seem like a good solution

Reply:
Basically yes: https://github.com/chanzuckerberg/idseq-web/blob/master/app/services/retrieve_pipeline_viz_graph_data_service.rb#L238